invenio_app_rdm/config.py

```diff
@@ -652,6 +652,12 @@ def files_rest_permission_factory(obj, action):
 APP_RDM_DEPOSIT_FORM_PUBLISH_MODAL_EXTRA = ""
 """Additional text/html to be displayed in the publish and submit for review modal."""
 
+APP_RDM_DEPOSIT_REQUIRE_FILES = False
+"""Enable/disable metadata-only records"""
+
+APP_RDM_DEPOSIT_FULL_RECORD_VISIBILITY = True
```

Comment:
what about the backend validation? it is still possible to pass the access via REST API